src/main/java/io/github/ryotan/code/util/CodeEnumReflectionUtil.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * These codes are licensed under CC0.
+ *
+ * http://creativecommons.org/publicdomain/zero/1.0/deed
+ * http://creativecommons.org/publicdomain/zero/1.0/deed.ja
+ */
+
+package io.github.ryotan.code.util;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+import io.github.ryotan.code.CodeEnum;
+import io.github.ryotan.code.CodeEnum.Filter;
+import io.github.ryotan.code.CodeEnum.ShortLabel;
+
+
+/**
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [First sentence of Javadoc is incomplete (period is missing) or not present.](https://www.codacy.com/app/ryotan/code-enum-support/file/3991998003/issues/source?bid=3375378&fileBranchId=3375797#l26)